### ISSUE TITLE
Inconsistency: which remote access action is available on secure screens?

### ISSUE THREAD

**Issue by CyrilleB79** [nvaccess/nvda#17945]
### Steps to reproduce:

* Enable Remote access feature in Remote Access settings
* Restart NVDA in secure mode, i.e. executing "nvda -s"
* Press NVDA+alt+R
* Open NVDA menu -> Tools -> Remote -> Connect

### Actual behavior:
Using `NVDA+alt+r` gesture, NVDA indicates me that this action is unavailable in a secure context.
But going through NVDA menu, I can open the "Connect" dialog.

### Expected behavior:
Both actions should be consistent, i.e. either opening the Connect dialog should be allowed or not, but it should not matter how you try to open it.

Also, it should be clarified in the User Guide what is available on secure screens / in secure NVDA or not:
* Can you initiate a connecion to control another computer?
* Can you only initiate a connection if you are the controlled machine?
* Or should all kinds of manual connection be disabled, i.e. only automatic connection to take into account UAC screens on the controlled computer?
* Are all types of automatic connection allowed on secure screens? Or only if you are the controlled machine?
* Is hosting locally a server allowed on secure screen?

If any of these possibilities differ from the normal (non secure screen) use case, the User Guide shoudl make this explicit.

#### NVDA installed/portable/running from source:
From source
#### NVDA version:
Last beta (commit 9833d781273eeb6f4891e2cba8e110cbf4e725af)

#### Windows version:
Windows 10 22H2 (AMD64) build 19045.5608
#### Name and version of other software in use when reproducing the issue:
N/A
#### Other information about your system:
N/A
#### Does the issue still occur after restarting your computer?
Not tested
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Not tested
